### ISSUE TITLE
improve accessibility

### ISSUE THREAD

**Issue by gabecar** [gabecar/scrushable#56]
The accessibility of `SelectSearchable` needs to be improve
ex:
![image](https://user-images.githubusercontent.com/23194736/138367883-e177469b-7556-4ffb-be69-ba21b8e82b16.png)
this aria-label should be passed as prop
